iree/samples/static_library/README.md

```diff
@@ -42,7 +42,7 @@ for general instructions on building using CMake):
   cmake -B ../iree-build/
     -DIREE_BUILD_SAMPLES=ON \
     -DIREE_TARGET_BACKENDS_TO_BUILD=DYLIB-LLVM-AOT \
-    -DIREE_HAL_DRIVERS_TO_BUILD=DYLIB \
+    -DIREE_HAL_DRIVERS_TO_BUILD=Dylib_Sync \
```

Comment:
DYLIB_SYNC for consistency (even though case insensitive)

Reply:
What about the main `CMakeLists.txt`?
https://github.com/google/iree/blob/11d6202b2562601348d08d14c2ff807fcfe70b6f/CMakeLists.txt#L96-L103